README.md

```diff
@@ -93,6 +93,7 @@ Run `make help` to see all the possible targets and variables:
     LEPTONICA_VERSION  Leptonica version. Default: 1.78.0
     TESSERACT_VERSION  Tesseract commit. Default: 4.1.0
     TESSDATA_REPO      Tesseract model repo to use. Default: _best
+    TESSDATA           Name of the traineddata file to start finetuning from (without file extension)
```

Comment:
The name of the traineddata file is `STARTMODEL.traineddata`.

`TESSDATA` is the name of the directory of the traineddata file.

Reply:
If we suggest `TESSDATA` as a variable which may be set by the user, that should also be documented in the `Makefile`.

Reply:
Done! Let me know if it's ok